### PR TITLE
[chore] Release 2.0.0

### PR DESCRIPTION
**Pull request by LukasSanda**
## 🏷 Description

- Release 2.0.0

### Features
- Add Live chat mode
- Minimum iOS deployment target set to iOS 14.0
- Add localized strings to sdk package
- Disable modification for archived thread
- implement file restrictions parsing and publication
- Deprecate legacy plugins
- Handle sent message via `onThreadUpdated(_:)`
- Privacy Manifest

### Bug Fixes
- Rotation bug on iOS15
- Incorrect attachment size displayed for single attachment
- append UTType.movie for all videos
- Update ListFieldView behavior for iOS 14
- Exclude links from multiattachments view
- remove validation for Prechat survey
- 
## 🛠 What's missing?

- None

## 📋 Checklist

- [x] Checks - Merge only when all tests passed and there are not SwiftLint warnings.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,31 @@
...
+
+### Bug Fixes
+- Rotation bug on iOS15
+- Incorrect attachment size displayed for single attachment
+- append UTType.movie for all videos
+- Update ListFieldView behavior for iOS 14
+- Exclude links from multiattachments view
+- remove validation for Prechat survey
+
+
```
Comment (pavelreiter) [inline CHANGELOG.md]: Nit:
extra newline, can be removed

File: README.md [hunk 2]
```diff
@@ -5,24 +5,6 @@
...
-
-### Swift Package Manager
-
-You can use Swift Package Manager to add CXoneChatUI to your Xcode project. Simply follow these steps:
-
-1. Open your Xcode.
-2. Navigate to `File > Swift Packages > Add Package Dependency...`
-3. Enter the SDK repository URL https://github.com/nice-devone/nice-cxone-mobile-ui-ios in the search bar.
-4. Select the version or branch you want to use.
-
```
Comment (pavelreiter) [inline README.md]: Sorry for the question, but why was this removed?

Comment (LukasSanda) [reply README.md]: It has been overridden automatically with GitHub action, good catch

**Review by pavelreiter — APPROVED** [review]
LGTM
